conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureUndertowRuntime.java

```diff
@@ -39,7 +39,9 @@
 
     private ConjureUndertowRuntime(Builder builder) {
         this.bodySerDe = new ConjureBodySerDe(
-                builder.encodings.isEmpty() ? ImmutableList.of(Encodings.json(), Encodings.cbor()) : builder.encodings);
+                builder.encodings.isEmpty()
+                        ? ImmutableList.of(Encodings.json(), Encodings.smile(), Encodings.cbor())
```

Comment:
Would be cool to wrap these in instrumented encodings so we can capture which encoding is being used when

Reply:
I like it, I'll implement something similar to the traced-encoding wrapper in a follow up